packages/@jsii/go-runtime/jsii-runtime-go/api.go

```diff
@@ -89,6 +89,11 @@ type objref struct {
 	InstanceID string `json:"$jsii.byref"`
 }
 
+// EnumRef is a reference to a jsii enum member
+type EnumRef struct {
+	MemberFqn string `json:"$jsii.enum"`
```

Comment:
Let's try to be consistent with go local naming style, which has acronyms spelled out ALL UPPERCASE.

```suggestion
	MemberFQN string `json:"$jsii.enum"`
```

Reply:
Done